core/jazz-ui/src/app/secondary-components/create-service/internal/create-service.component.ts

```diff
@@ -108,7 +108,7 @@ export class CreateServiceComponent implements OnInit {
   model = new ServiceFormData('', '', '', '', '', '');
   cronObj = new CronObject('0/5', '*', '*', '*', '?', '*')
   rateExpression = new RateExpression(undefined, undefined, 'none', '5', this.selected, '');
-  eventExpression = new EventExpression("awsEventsNone", undefined, undefined, undefined);
+  eventExpression = new EventExpression("awsEventsNone", undefined, undefined, undefined, undefined);
```

Comment:
why do you need to keep adding undefined. Just doesn't look pleasing

Reply:
I think this is out of the scope of the current story. The EventExpression class should be using optional parameters or not be used at all. But we can decide that at a later date.

Reply:
well my comment is nothing to do with whether its internal or oss but more to do with code quality.